package.json

```diff
@@ -54,6 +54,7 @@
     "plur": "^3.0.1",
     "portfinder": "^1.0.19",
     "querystring": "^0.2.0",
+    "redis": "^2.8.0",
```

Comment:
Merge the latest `sp_multi` in your branch as I've added this dependency as `optional` already